lib/_sendEvent.ts

```diff
@@ -10,7 +10,7 @@ declare var process : {
 
 export type AnalyticsEvent = {
   userID: string;
-  timestamp: number;
+  timestamp?: number;
```

Comment:
Why do we want to keep the `timestamp` optional since we don't use it anymore?

Reply:
My intention was that timestamp can still provided, only now we don't override it with `Date.now()`.